src/global_vars.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Needs to be in modules after all (I'm very sorry!), I am unable to import. Either one is fine:
`src/modules/global_vars.py` -> `from ..global_vars import APPLICATION_DIR, CONFIG_RULES`
`src/modules/global_vars/__init__.py` -> `from modules.global_vars import APPLICATION_DIR, CONFIG_RULES`